dal/RestDB.py

```diff
@@ -0,0 +1,53 @@
+from __future__ import absolute_import
+from tinydb import TinyDB, Query
+from dal.BaseDB import BaseDB
+from typing import List
+from typing import Dict
+
+
+class RestDB(BaseDB):
+    db = TinyDB('db.json')
```

Comment:
should not hardcode the filename. It should be passed from app.py through a constructor. Which should, in turn, fetch from config file. 
@mansikulkarni96